front-end/src/util/__tests__/formatBnBalance.test.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import formatBnBalance from '../formatBnBalance';
+
+jest.mock('../getNetwork', () => jest.fn(() => {return 'polkadot';}));
+const getNetwork = require('../getNetwork');
+
+describe('Formatting BN Balances', () => {
+	beforeEach(() => {
+		jest.resetModules();
+	});
+
+	const formattedBalance1 = '1.00 ';
```

Comment:
This would also fail without trailing space

Reply:
Same, this should be fixed.